### PR TITLE
#182282679 - CONFIGURE POSTGRESQL AS THE DATABASE ENGINE.

### PR DESCRIPTION
**Pull request by dalienst**
#### What does this Pull Request do?
Configure Speaksfer to use PostgreSQL as DB Engine
#### Description of Task to be completed?
Configuration of PostgreSQL as the Database engine of the project.
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
The application needs a database to store data.
#### What are the relevant Pivotal Tracker?
[#182282679 ](https://www.pivotaltracker.com/n/projects/2570600/stories/182282679)
#### Questions:
None

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+Django = "4.0.5"
```
Comment (mainamuriuki274) [inline Pipfile:7]: should be ==4.0.5

File: speaksfer/settings/__init__.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from .base import *
+
+
+if config("ENV_NAME") == 'Production':
+    from .production import *
+elif config("ENV_NAME") == 'Staging':
+    from .staging import *
+else:
+    from .local import *
```
Comment (mainamuriuki274) [inline speaksfer/settings/__init__.py:9]: add an empty line after this line

File: speaksfer/settings/local.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from speaksfer.settings.base import ALLOWED_HOSTS
+
+
+DEBUG = True
+
+ALLOWED_HOSTS += [
+    'localhost',
+    '127.0.0.1'
+]
```
Comment (mainamuriuki274) [inline speaksfer/settings/local.py:9]: add an empty line after this file

File: speaksfer/settings/production.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from speaksfer.settings.base import ALLOWED_HOSTS
+
+
+DEBUG = False
+
+ALLOWED_HOSTS += [
+    'localhost',
+    '127.0.0.1'
+]
```
Comment (mainamuriuki274) [inline speaksfer/settings/production.py:9]: Remove ALLOWED HOSTS in this file for now

File: speaksfer/settings/staging.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+from speaksfer.settings.base import ALLOWED_HOSTS
+
+
+DEBUG = True
+
+ALLOWED_HOSTS += [
+    'localhost',
+    '127.0.0.1'
+]
```
Comment (mainamuriuki274) [inline speaksfer/settings/staging.py:9]: Remove ALLOWED HOSTS in this file for now

**Review by mainamuriuki274 — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (dalienst) [reply Pipfile]: Okay

File: speaksfer/settings/__init__.py (hunk 2 above)
Comment (dalienst) [reply speaksfer/settings/__init__.py]: Done

**Review by mainamuriuki274 — CHANGES_REQUESTED** [review]

File: speaksfer/settings/__init__.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+from decouple import config
+
+from .base import *  # noqa F403, F401
+
+if config("ENV_NAME") == "Production":  # noqa F403, F401
```
[the remark this answers is not in the document]
Comment (dalienst) [reply speaksfer/settings/__init__.py]: All changes made

File: requirements.txt [hunk 7]
```diff
@@ -0,0 +1,8 @@
+asgiref==3.5.2
```
[the remark this answers is not in the document]
Comment (dalienst) [reply requirements.txt:1]: Done

File: Pipfile [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+name = "pypi"
+
+[packages]
+dj-database-url = "==0.5.0"
+psycopg2-binary = "==2.9.3"
+python-decouple = "==3.6"
+django = "==4.0.5"
+
+[dev-packages]
+pre-commit = "*"
```
[the remark this answers is not in the document]
Comment (dalienst) [reply Pipfile]: Done

**Review by mainamuriuki274 — APPROVED** [review]

**Pull request merged**